server.py

```diff
@@ -206,8 +206,15 @@ def serve_media(filename):
 @schema.validate(connectionRequestsSchema)
 def createConnectionRequest():
     connectionRequests.makeRequest(getRequesterIdInt(), request.json.get('requesteeID'), request.json.get('message'))
-    return jsonMessageWithCode('')
-#new
+    return jsonMessageWithCode('success')
+
+@app.route('/api/connection-requests/<int:connectionRequestId>/resolved', methods=['POST']) #is post correct?
```

Comment:
Here we don't use post method, we don't submit any information to the server.

Reply:
Okay! What sort of request is it then!? And how are we changing the information (f to t) without this request?!

Reply:
Actually, I think the route should be: `/api/connection-requests/<int:userId>/<int:connectionRequestId/resolved`, because we need to make sure the operator is the admin. Like Kent's PR, you can see the example. As long as we go to the URL: localhost:5000/api/connection-requests/1/1/resolved, function: resolveConnectionRequest will be called in server.py, then it will call handler and database corresponding functions to change value in database.

Reply:
I had this but I ended it up taking it out after talking to Graham because I think he changed part of the code that enforced the user ID. Maybe we can ask him about this today.